database/src/main/resources/db/migration/V1.83.0__RHCLOUD-28465_drop_old_email_subscription_table.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE endpoint_email_subscriptions;
```

Comment:
I would wait for at least 2 weeks between the Java code removal and the DB table drop. Just in case we would need to rollback the Java changes for some reason 😃

If you agree with that, please move the SQL file to a different PR that will be merged later.

Reply:
Yes, good point